apps/src/sites/studio/pages/projects/public.js

```diff
@@ -33,7 +35,9 @@ $(document).ready(() => {
             canViewFullList
             canViewAdvancedTools
           />
-          <PublicGallery />
+          <PublicGallery
+            limitedGallery={projectsData.limitedGallery}
```

Comment:
especially because a falsy value is dangerous, this should be a required prop in PublicGallery. I'm not sure any test would have failed as a result in this case, though.